content-src/lib/selectLayoutRender.js

```diff
@@ -2,7 +2,12 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-export const selectLayoutRender = (state, prefs, rickRollCache) => {
+export const selectLayoutRender = ({
+  state = {},
+  prefs = {},
+  rollCache = [],
```

Comment:
Just changing this little bit of whimsical naming, mostly because now it's exposed as an actual option value in function call.